.travis.yml

```diff
@@ -130,41 +130,86 @@ jobs:
     - <<: *js
       stage: Test
       env: [{task: "js:flow"}]
+      before_script:
+        - echo "loglevel=silent" >> .npmrc
+        - mkdir -p logs/
+        - set -o pipefail
+        - npm run report $task "pending" "Checking the flow of things..."
```

Comment:
If we move this into the `script` section itself (eww, I know), we could remove these other 4 lines that you had to add to each job… just an idle thought.

Reply:
Yeah, that's probably a good idea. Putting it as the first key will ensure it gets run